tests/integration/Tribe/Admin/__snapshots__/OrderReportTest__test_tc_order_report_display__event with multiple tickets in same order__0.snapshot.html

```diff
@@ -131,11 +131,11 @@ <h3>Sales Totals</h3>
 			<option value="id">
 			Search by Order ID		</option>
 	</select><input type="submit" id="search-submit" class="button" value="Search Orders"  /></p>
-		<input type="hidden" id="_wpnonce" name="_wpnonce" value="1234567890" /><input type="hidden" name="_wp_http_referer" value="" />	<div class="tablenav top">
+		<input type="hidden" id="_wpnonce" name="_wpnonce" value="1234{{ID}}90" /><input type="hidden" name="_wp_http_referer" value="" />	<div class="tablenav top">
 
 				<div class="alignleft actions bulkactions">
 					</div>
-					<div class="alignleft actions attendees-actions"><input type="button" name="print" class="print button action" value="Print"><a target="_blank" href="?orders_csv=1&#038;orders_csv_nonce=1234567890" class="export action button" rel="noopener noreferrer">Export</a></div>
+					<div class="alignleft actions attendees-actions"><input type="button" name="print" class="print button action" value="Print"><a target="_blank" href="?orders_csv=1&#038;orders_csv_nonce=1234{{ID}}90" class="export action button" rel="noopener noreferrer">Export</a></div>
```

Comment:
These snapshot updates also dont look correct. Why is the {{ID}} placeholder in the center of where the nonce is supposed to be ?

Reply:
It would seem that it's due to a find/replace in the test itself that looks for post IDs and replaces them with `{{ID}}`. It would see that `5678` is one of the post IDs.

Reply:
Can you change the return value of the wp_create_nonce for the specific test ?